commons/src/lib.rs

```diff
@@ -172,19 +196,44 @@ mod tests {
     #[test]
     fn test_validate_content_type() {
         let mut headers = actix_web::http::HeaderMap::new();
-        validate_content_type(&headers, "application/json").unwrap(); // if the request leaves Accept empty, we can return whatever we want
+
+        let content_type: Vec<actix_web::http::HeaderValue> =
+            vec![header::HeaderValue::from_str("application/json").unwrap()];
+        let version = validate_content_type(&headers, content_type.clone()).unwrap(); // if the request leaves Accept empty, we return the minimum supported cincinnati version
+        assert_eq!(version, MIN_CINCINNATI_VERSION.to_string());
+
         headers.insert(
             header::ACCEPT,
             //"application/json, text/*; q=0.2".parse().unwrap(), // prefer JSON, but also accept any text/* after an 80% markdown in quality.  FIXME: needs a smarter parser in validate_content_type
             "application/json".parse().unwrap(),
         );
-        validate_content_type(&headers, "application/json").unwrap();
+        let version = validate_content_type(&headers, content_type.clone()).unwrap();
+        assert_eq!(version, "application/json");
+
         headers.insert(
             // FIXME: drop once validate_content_type gets a smarter parser and the previous insert can include the text/* entry
             header::ACCEPT,
             "text/*".parse().unwrap(),
         );
-        validate_content_type(&headers, "text/plain").unwrap();
-        validate_content_type(&headers, "image/png").unwrap_err();
+        let text_type: Vec<actix_web::http::HeaderValue> =
+            vec![header::HeaderValue::from_str("text/plain").unwrap()];
+        let version = validate_content_type(&headers, text_type).unwrap();
+        assert_eq!(version, "text/*");
+
+        let image_type: Vec<actix_web::http::HeaderValue> =
+            vec![header::HeaderValue::from_str("image/png").unwrap()];
+        validate_content_type(&headers, image_type).unwrap_err();
+
+        headers.insert(
+            // FIXME: drop once validate_content_type gets a smarter parser and the previous insert can include the text/* entry
+            header::ACCEPT,
+            "application/vnd.redhat.cincinnati.v1+json".parse().unwrap(),
+        );
+        let cincinnati_v1: Vec<actix_web::http::HeaderValue> =
+            vec![
+                header::HeaderValue::from_str("application/vnd.redhat.cincinnati.v1+json").unwrap(),
```

Comment:
can we also accept `application/json` here?  Because in production, the server will support both `application/json` (for backwards-compat with existing CVOs) and this versioned type, and we'll want to ensure that we respond with `application/json` to those older CVOs and `application/vnd.redhat.cincinnati.v1+json` to newer CVOs out of the same server config.

Reply:
ok, `validate_content_type(&headers, cincinnati_v1, accept_default.clone())` below now, and the default type gets injected into the allowed list, so I'm good here.

Reply:
```suggestion
            MIN_CINCINNATI_VERSION.to_string().parse().unwrap(),
```
less hardcodes